INSTALL_FROM_GITHUB.md

```diff
@@ -0,0 +1,40 @@
+## How to compile openFrameworks from github
+
+This instructions are useful just in case you want to submit a pull request or you want to have always the last openFrameworks version. In any other case, most probably, you don't need to clone the entire openFrameworks repository.
+
+If you are a developer, or if you want to submit a pull request, [read this first](https://github.com/openframeworks/openFrameworks/blob/master/README.md#developers)
+
+Let's start cloning the last `stable` branch of openframeworks.
+
+```bash
+git clone git@github.com:openframeworks/openFrameworks.git
+```
+
+As the external dependencies are not in the repository, you need to download them. To make the things simplier, there is a bash script for this goal called `download_lib.sh`. This bash script is located in the `scripts` folder. In this folder, there are several subfolders, one for each paltform. Assuming you are, for example, using OSX, you need to run `/bin/bash scripts/osx/download_libs.sh`.
+
+Now you have all the files that you need, you just need to compile them and to setup all the projects contained in the `examples` folder. The easiest way to do this is to use the projectGenerator to compile all the examples.
+The projectGenerator is useful app to create new project and to add and remove addons. As it is a separate project, with a well defined scope, the openFrameworks developer decided to put it in a separate repository. You have 2 ways to get the project generator in your openFrameworks installation:
+* You can initialize and update all the git submodules (including the projectGenerator) running first `git submodule init` and then `git submodule update`
+* Or, more easily, you can download an already packaged openFrameworks installation from the [download page](http://openframeworks.cc/download/), and copy the `projectGenerator` folder in your openFrameworks installation.
+
+As the first option can lead to [unexpected results](https://forum.openframeworks.cc/t/how-to-build-project-generator-from-of-git-repo/26232), we suggest you the second one.
+
+Now that you have the project generator, you are ready to compile all the examples. Do as follow:
+
```

Comment:
I see. But the instructions you have linked explains how to use the project generator to create one project. Not how to compile all the examples. Maybe I should move this part to the page you have linked?